### PR TITLE
Add 'My Imports & Exports' to My Reading Log

### LINKED ISSUE #4795: Add "Import and Export Options" to My Profile
### Describe the problem that you'd like solved

Being an OL member for the past 5 months I was not aware exactly where this option was until today I found it in Settings.
I believe this feature should be added to the More menu / My profile so that more users will come to know about this.

### Stakeholders

@mekarpeles

### LINKED ISSUE #4987: Add to list button is too big on page of orphaned book
The add to list button is very big on this page with an orphaned book. Probably related to #3561

### Evidence / Screenshot (if possible)
image

### Relevant url?
https://openlibrary.org/books/OL9475434M/Pokemon_The_First_Movie_Animation_Comics?edition=

### Steps to Reproduce

1. Go to the page

* Actual: Button expands beyond parent div
* Expected: Button stays in parent div

### Details

- **Logged in (Y/N)?** Y
- **Browser type/version?** Brave
- **Operating system?** Mac
- **Environment (prod/dev/local)?** prod

### PR DESCRIPTION
**Pull request by Yashs911**
Closes #4795 ; Closes #4987

Added My Imports & Exports to My Profile.
Fixed Add to List button is too big

### Screenshot

image
### Stakeholders

@jamesachamp

### CODE REVIEW

**Comment by mekarpeles** [comment]
@cdrini mentioned that moving the exports option to the reading log / lists may be better than the account drop down, but the idea of making this feature more accessible sounds great

**Comment by Yashs911** [comment]
@mekarpeles Will this work?
image

File: static/css/components/dropper.less [hunk 1]
```diff
@@ -195,7 +195,7 @@ div.Tools {
   // And Orphaned Editions (no work)
   .widget-add.old-style-lists {
     .dropclick {
-      padding: 10px;
+      padding: 5px;
```
Comment (jamesachamp) [inline static/css/components/dropper.less:198]: Is this needed?  This seems to be unrelated to this PR.

File: openlibrary/templates/account/books.html [hunk 2]
```diff
@@ -100,6 +100,7 @@ <h3>
                 <li><a class="list-link" href="$(lst.key)">$lst['name'] ($(len(lst.seeds)))</a></li>
             </div>
           </ul>
+        <p class="sansserif large"><a href="/account/import">$_("Import & Export Options")</a></p>
```
Comment (jamesachamp) [inline openlibrary/templates/account/books.html:103]: To be consistent with the other sidebar sections. I think that the import/export section of the sidebar should be an unordered list with a subsection list item.  The subsection could say something like "Import / Export".  It would be good if the new `<ul>` element had a unique class name ending with `-lists`, but used the same CSS rules as the `user-lists` class.

Keeping all of these sections structured similarly will make things simpler when and if we convert this to a Vue component in the future.

**Review by jamesachamp — CHANGES_REQUESTED** [review]
Thanks @Yashs911!  I think that this is a great place to link to our import and export options.  Will happily approve once the suggested changes are made.

File: static/css/components/dropper.less (hunk 1 above)
Comment (Yashs911) [reply static/css/components/dropper.less:198]: It's related to the other issue this PR closes https://github.com/internetarchive/openlibrary/issues/4987.

File: openlibrary/templates/account/books.html (hunk 2 above)
Comment (Yashs911) [reply openlibrary/templates/account/books.html]: Sorry for the delay on this one.
I have updated the PR.
image

File: openlibrary/templates/account/books.html [hunk 3]
```diff
@@ -100,6 +100,9 @@ <h3>
                 <li><a class="list-link" href="$(lst.key)">$lst['name'] ($(len(lst.seeds)))</a></li>
             </div>
           </ul>
+        <ul class="import-export-lists">
+          <li class="subsection"><a href="/account/import">$_("Import / Export")</a></li>
```
Comment (jamesachamp) [inline openlibrary/templates/account/books.html:104]:
```suggestion
          <li class="subsection">$_("Import / Export")</li>
          <li><a href="/account/import">$_("Import and export options")</a></li>
```
Prefer having the link not also be the subsection header.  The link should also have `text-decoration: underline` to indicate that patrons will be navigating away from the page when clicking the link.

**Review by jamesachamp — CHANGES_REQUESTED** [review]
Thanks again!  Just one more small change.

File: openlibrary/templates/account/books.html (hunk 3 above)
Comment (Yashs911) [reply openlibrary/templates/account/books.html]: PR updated
image

**Review by jamesachamp — APPROVED** [review]

**Comment by mekarpeles** [comment]
throwing on testing.

The position of the links seems a tiny strange to me (Reading Logs & Lists in the sidebar are all types of collections you can explore, whereas the import/exports seem more similar to managing your preference.

I'm happy for it to land as it, its positioning may get tweaked a bit in a future PR, thanks for the hard work!
